internal/infra/pull/sql_datasource.go

```diff
@@ -81,7 +81,7 @@ func (ds *SQLDataSource) RowReader(source pull.Table, filter pull.Filter) (pull.
 	sql.Write([]byte("SELECT * FROM "))
 	sql.Write([]byte(ds.tableName(source)))
 	sql.Write([]byte(" "))
-	if len(filter.Values()) > 0 {
+	if len(filter.Values()) > 0 || filter.Where() != "" {
```

Comment:
Maybe use [a TrimSpace](https://golang.org/pkg/strings/#TrimSpace) here instead of `filter.Where() != ""` (or in a preventive way at start of the method)